repo_e/capital_letter.py

```diff
@@ -0,0 +1,7 @@
+def capital(strings):
+    Capital_str=[]
```

Comment:
please stick to snake_case naming, so `capital_str` instead of `Capital_str` (this is a mix of snake_case and CamelCase). Also - don't encode variable type in it's name, don't put `str`, `int`, `float`, `list` to names of variables